packages/create-amplify/src/npm_package_manager_controller.ts

```diff
@@ -28,9 +30,27 @@ export class NpmPackageManagerController implements PackageManagerController {
     if (type === 'dev') {
       args.push('--save-dev');
     }
-    await this.execa(this.executableName, args, {
-      stdio: 'inherit',
-      cwd: this.projectRoot,
-    });
+
+    let aggregatedStdout = '';
+    const aggregatorStream = new stream.Writable();
+    aggregatorStream._write = function (chunk, encoding, done) {
```

Comment:
instead of concatenating all of this in memory, could we instead pipe directly to logger.debug? That way if you have the --debug flag set, you get live logs rather than having to wait for the whole thing to finish and then getting a wall of text

Reply:
Addressed as part of change in [0735be9](https://github.com/aws-amplify/amplify-backend/pull/595/commits/0735be9988ffbeba2b82c2c3be25e57a1adae09e)